_examples/golang-imports/api.gen.go

```diff
@@ -84,6 +84,14 @@ func (x *Location) Is(values ...Location) bool {
 	return false
 }
 
+var (
+	methodAnnotations = map[string]map[string]string{
+			"/rpc/ExampleAPI/Ping": {},
+			"/rpc/ExampleAPI/Status": {},
+			"/rpc/ExampleAPI/GetUsers": {},
```

Comment:
@LukasJenicek I'm thinking we could render the whole `MethodCtx{}` here:

```go
var methodCtx = map[string]MethodCtx{
	"/rpc/ExampleAPI/Ping": {Name: "Ping", Service: "ExampleAPI", Annotations: {}},
	"/rpc/ExampleAPI/Status": {Name: "Status", Service: "ExampleAPI", Annotations: {"internal"}},
	"/rpc/ExampleAPI/GetUsers": {Name: "GetUsers", Service: "ExampleAPI", Annotations: {}},
}
```

And then `MethodFromContext()` would return its value.